spec/jobs/set_governing_apo_job_spec.rb

```diff
@@ -96,34 +96,30 @@
           )
         end
 
-        let(:object_client1) { instance_double(Dor::Services::Client::Object, find: cocina1) }
+        let(:object_client1) { instance_double(Dor::Services::Client::Object, find: cocina1, update: true) }
         let(:object_client3) { instance_double(Dor::Services::Client::Object, find: cocina3) }
 
         before do
           allow(Dor::Services::Client).to receive(:object).with(pids[0]).and_return(object_client1)
           allow(Dor::Services::Client).to receive(:object).with(pids[1]).and_raise(Dor::Services::Client::NotFoundResponse)
           allow(Dor::Services::Client).to receive(:object).with(pids[2]).and_return(object_client3)
 
-          allow(Dor).to receive(:find).with(pids[0]).and_return(item1)
+          # allow(Dor).to receive(:find).with(pids[0]).and_return(item1)
```

Comment:
Do we want to keep these comments for now or cruft?

Reply:
It's cruft. I've removed.